lib/pbio/drv/usb/stm32_usbd/usbd_pybricks.h

```diff
@@ -0,0 +1,133 @@
+/**
+  ******************************************************************************
+  * @file    usbd_pybricks.h
+  * @author  MCD Application Team
+  * @brief   Header file for the usbd_pybricks.c file.
+  ******************************************************************************
+  * @attention
+  *
+  * Copyright (c) 2015 STMicroelectronics.
+  * All rights reserved.
+  *
+  * This software is licensed under terms that can be found in the LICENSE file
+  * in the root directory of this software component.
+  * If no LICENSE file comes with this software, it is provided AS-IS.
+  *
+  ******************************************************************************
+  */
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __USB_PYBRICKS_H
+#define __USB_PYBRICKS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Includes ------------------------------------------------------------------*/
+#include  "usbd_ioreq.h"
+
+/** @addtogroup STM32_USB_DEVICE_LIBRARY
+  * @{
+  */
+
+/** @defgroup USBD_Pybricks
+  * @brief This file is the header file for usbd_pybricks.c
+  * @{
+  */
+
+
+/** @defgroup USBD_Pybricks_Exported_Defines
+  * @{
+  */
+#define USBD_MS_VENDOR_CODE                0xCC
+#define USBD_SIZ_MS_OS_DSCRPTR_SET         (10 + 20 + 130)
+
+#define USBD_PYBRICKS_CONFIG_DESC_SIZ      (9 + 9 + 7 + 7)
+
+#define USBD_PYBRICKS_IN_EP                0x81U  /* EP1 for data IN */
+#define USBD_PYBRICKS_OUT_EP               0x01U  /* EP1 for data OUT */
+
+#define USBD_PYBRICKS_MAX_PACKET_SIZE      64U
```

Comment:
I think most of these are only used in one file (especially if we move `USBD_OSDescSet`) and can be defined in the .c file rather than in the .h file.

Reply:
I could be mistaken, but I think we resolved to leave this here.